llvm/test/MC/AMDGPU/gfx12_asm_vdsdir.s

```diff
@@ -1,103 +1,103 @@
 // RUN: llvm-mc -arch=amdgcn -mcpu=gfx1200 -show-encoding %s | FileCheck -check-prefix=GFX12 %s
 
 ds_direct_load v1 wait_va_vdst:15
-// GFX12: ds_direct_load v1 wait_va_vdst:15  ; encoding: [0x01,0x00,0x1f,0xce]
+// GFX12: ds_direct_load v1 wait_va_vdst:15 wait_vm_vsrc:0 ; encoding: [0x01,0x00,0x1f,0xce]
 
 ds_direct_load v2 wait_va_vdst:14
-// GFX12: ds_direct_load v2 wait_va_vdst:14  ; encoding: [0x02,0x00,0x1e,0xce]
+// GFX12: ds_direct_load v2 wait_va_vdst:14 wait_vm_vsrc:0 ; encoding: [0x02,0x00,0x1e,0xce]
 
 ds_direct_load v3 wait_va_vdst:13
-// GFX12: ds_direct_load v3 wait_va_vdst:13  ; encoding: [0x03,0x00,0x1d,0xce]
+// GFX12: ds_direct_load v3 wait_va_vdst:13 wait_vm_vsrc:0 ; encoding: [0x03,0x00,0x1d,0xce]
 
 ds_direct_load v4 wait_va_vdst:12
-// GFX12: ds_direct_load v4 wait_va_vdst:12  ; encoding: [0x04,0x00,0x1c,0xce]
+// GFX12: ds_direct_load v4 wait_va_vdst:12 wait_vm_vsrc:0 ; encoding: [0x04,0x00,0x1c,0xce]
 
 ds_direct_load v5 wait_va_vdst:11
-// GFX12: ds_direct_load v5 wait_va_vdst:11  ; encoding: [0x05,0x00,0x1b,0xce]
+// GFX12: ds_direct_load v5 wait_va_vdst:11 wait_vm_vsrc:0 ; encoding: [0x05,0x00,0x1b,0xce]
 
 ds_direct_load v6 wait_va_vdst:10
-// GFX12: ds_direct_load v6 wait_va_vdst:10  ; encoding: [0x06,0x00,0x1a,0xce]
+// GFX12: ds_direct_load v6 wait_va_vdst:10 wait_vm_vsrc:0 ; encoding: [0x06,0x00,0x1a,0xce]
 
 ds_direct_load v7 wait_va_vdst:9
-// GFX12: ds_direct_load v7 wait_va_vdst:9   ; encoding: [0x07,0x00,0x19,0xce]
+// GFX12: ds_direct_load v7 wait_va_vdst:9  wait_vm_vsrc:0 ; encoding: [0x07,0x00,0x19,0xce]
```

Comment:
Nit: it is a bit strange putting two spaces before "wait_vm_vsrc". Some dasm tests run `FileCheck -strict-whitespace` to avoid this kind of thing.